### DISCUSSION TITLE
How do I tweak glues?

### DISCUSSION THREAD

**Discussion by neruthes** [typst/typst#3961] [Q&A] [answered]
I traditionally use xeCJK with `CJKglue=\hskip 0pt,CJKecglue=\hksip 5pt plus 100pt minus 0.5pt` to make sure that in non-ragged paragraphs the inline-stretching duty is solely spread among CJK to non-CJK glues while CJK to CJK glues are always `\hskip 0pt`. I may sometimes prefer other glue configurations regarding the nature of particular documents.

I hope that shifting to Typst won't break my workflows and working with glues is an important consideration. However I searched all around and could not find any mention of glues in Typst. I wonder if this is a terminology mismatch, a documentation miss, or a functionality shortage. Can someone lead the way to the right place in documentation? Or confirm that my workflow cannot be reproduced with Typst?

**Comment by Enter-tainer** [comment, marked as answer]
> However I searched all around and could not find any mention of glues in Typst.

there is no glue in typst afaik
